src/etc/htmldocck.py

```diff
@@ -489,6 +491,19 @@ def check_command(c, cache):
                 found = get_tree_count(cache.get_tree(c.args[0]), c.args[1])
                 cerr = "Expected {} occurrences but found {}".format(expected, found)
                 ret = expected == found
+            elif len(c.args) == 4:  # @count <path> <pat> <text> <count> = count test
```

Comment:
This introduces a new type of test to the testing language. Where should that be documented?

Reply:
Also, some of the code below duplicates the code above for "at least one" matching. Can you extract them into a function to reduce repetition?

Reply:
> This introduces a new type of test to the testing language. Where should that be documented?

At the top of the file I think.